### PR TITLE
Add resource binding to website

### PR DESCRIPTION
**Pull request by ersin-ertan**

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -45,6 +45,14 @@ <h3 id="introduction">Introduction</h3>
   activity.title = (android.widget.TextView) activity.findViewById(2130968577);
 }</pre>
 
+            <h4 id="resource">Resource Binding</h4>
+            <p> Bind a pre-defined resources with <code>@BindBool, @BindColor, @BindDimen, @BindDrawable, @BindInt, @BindString</code>, which binds an <code>R.bool</code> ID(or your specificed type) to its corresponding field.</p>
```
Comment (JakeWharton) [inline website/index.html]: Remove leading space

Comment (JakeWharton) [inline website/index.html]: `<code>` and `</code>` should be around each `@BindWhatever` (not including the commas and spaces)

Comment (JakeWharton) [inline website/index.html]: Need a space before the parenthesis, also "specificed" should be "specified"

File: website/index.html [hunk 2]
```diff
@@ -45,6 +45,14 @@ <h3 id="introduction">Introduction</h3>
...
+            <pre class="prettyprint">class ExampleActivity extends Activity {
+  @BindString(R.string.title) String title;
+  @BindDrawable(R.drawable.graphic) Drawable graphic;
+  @BindColor(R.color.red) int red; // int or ColorStateList field
+  @BindDimen(R.dimen.spacer) Float spacer; // int (for pixel size) or float (for exact value) field</pre>
+
```
Comment (JakeWharton) [inline website/index.html]: Can you add:
```
  // ...
}
```
so that the class is complete?

**Comment by ersin-ertan** [comment]
Leave the trailing space that is prior to Non-Activity Binding?

File: website/index.html [hunk 3]
```diff
@@ -44,6 +44,16 @@ <h3 id="introduction">Introduction</h3>
...
 }</pre>
+            <h4 id="resource">Resource Binding</h4>
+            <p> Bind a pre-defined resources with <code>@BindBool</code>, <code>@BindColor</code>, <code>@BindDimen</code>, <code>@BindDrawable</code>, <code>@BindInt</code>, <code>@BindString</code>, which binds an <code>R.bool</code> ID(or your specified type) to its corresponding field.</p>
+            <pre class="prettyprint">class ExampleActivity extends Activity {
+  @BindString(R.string.title) String title;
+  @BindDrawable(R.drawable.graphic) Drawable graphic;
+  @BindColor(R.color.red) int red; // int or ColorStateList field
+  @BindDimen(R.dimen.spacer) Float spacer; // int (for pixel size) or float (for exact value) field
+  // ...
+  }
```
Comment (JakeWharton) [inline website/index.html]: Unindent this line. You can move `</pre>` up (see above code block).

File: website/index.html [hunk 4]
```diff
@@ -44,6 +44,16 @@ <h3 id="introduction">Introduction</h3>
   activity.footer = (android.widget.TextView) activity.findViewById(2130968579);
   activity.title = (android.widget.TextView) activity.findViewById(2130968577);
 }</pre>
+            <h4 id="resource">Resource Binding</h4>
+            <p> Bind a pre-defined resources with <code>@BindBool</code>, <code>@BindColor</code>, <code>@BindDimen</code>, <code>@BindDrawable</code>, <code>@BindInt</code>, <code>@BindString</code>, which binds an <code>R.bool</code> ID(or your specified type) to its corresponding field.</p>
```
Comment (JakeWharton) [inline website/index.html]: There's still a leading space after `<p>` and you need a space between "ID" and the opening paren `(`.

File: website/index.html (hunk 3 above)
Comment (ersin-ertan) [inline website/index.html]: I was wondering what line was 'this line' when 9 of them were selected, then looking at the above code block it was the '}' line XD

**Comment by ersin-ertan** [comment]
Also added the original carriage return which I thought was the leading space mentioned before and removed.

**Comment by JakeWharton** [comment]
Squashed and merged. Thanks!

**Comment by JakeWharton** [comment]
And deployed.

**Pull request closed without merging**